internal/cmd/get/get.go

```diff
@@ -43,6 +45,11 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
+func validateSemVer(versionString string) bool {
```

Comment:
I would simplify and remove this func. Then in 71 simply use `semver.STtrictVersion` with err handling.